nimble/host/mesh/src/model_cli.c

```diff
@@ -176,6 +176,8 @@ int bt_mesh_gen_onoff_set(u16_t net_idx, u16_t addr, u16_t app_idx,
 	}
 
 	if (!state) {
+		/* Increase transaction ID after sending set_unack message */
+		transaction_id++;
```

Comment:
Maybe we have increment it in `done:` label instead here and in status callback?

Reply:
Done, thanks!